tests try to generate tabcompletion

while running tests on jenkins for https://github.com/quattor/configuration-modules-core/pull/1036#issuecomment-283085607, i noticed following in the output
```
[ERROR] Failed to render tabcompletion: Failed to render with module tabcompletion: Non-existing template names: <undef> (abs_path of /usr/share/temp\
lates/quattor/CCM/tabcompletion.tt)
```
the used (o at least generated `ccm.cfg` file has `tabcompletion 0`, so this should not occur

--- a/build-scripts/src/main/perl/Test/Quattor/ProfileCache.pm
+++ b/build-scripts/src/main/perl/Test/Quattor/ProfileCache.pm
@@ -234,8 +234,8 @@ sub prepare_profile_cache
     my $ccmconfig = "$dirs->{resources}/ccm.cfg";
     if( ! -f $ccmconfig) {
         # Make a new default one
-        note("Creating default ccm.cfg in $dirs->{cache}");
-        $ccmconfig = "$dirs->{cache}/ccm.cfg";
+        note("Creating default ccm.cfg in $cache for profile $profile");
+        $ccmconfig = "$cache/ccm.cfg";
         if (! -f $ccmconfig) {
             my $fh = CAF::FileWriter->new($ccmconfig);
             print $fh get_ccm_config_default();

